oxidation/client/src/locales/fr-FR.json

```diff
@@ -92,9 +92,11 @@
         "about": "À propos",
         "support": "Aide et commentaires",
         "loggedIn": "Connecté",
-        "breadcrumbs": {
-            "documents": "Mes espaces"
-        }
+        "titles": {
+            "workspaces": "Mes espaces",
```

Comment:
```suggestion
            "workspaces": "Mes espaces de travail",
```

Reply:
Keeping it close to the mockups for now, easy to change later if we decide otherwise.